### PR TITLE
Os/wd 4555

### PR DESCRIPTION
**Pull request by OSoloviova**
Audit log page

### CODE REVIEW

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst [hunk 1]
```diff
@@ -0,0 +1,190 @@
+=========
+Audit Log
+=========
+
+The Audit Log is a feature that allows you to view and filter different events within a user's account.
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst:5]: replace "within a user's account" with "within an account"

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+It is located in the administrative panel
+
+.. image:: pic_auditLog/auditLogAdmin.png
+   :width: 600
+   :align: center
+
+Manage audit log
+================
+
+1. When you open the audit log, you will see all the events that occurred within the account on the current day for the current user.
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst:15]: "for the current user." is redundant information as it's not true and records will be written for all users who has access to the account

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+4. When the list is filtered by a specific parameter, you can view the following data:
+
+*Action* = Events that have occurred with various functionalities in the account, such as the creation of a dictionary, approval or signing of an envelope, creation of a new mailbox, and others listed above.
+*Error code* = An entry about errors that occurred, with the option to follow a link to documentation on error codes.
+*Source* = WEB or API.
+*Object* = Object name and its UUID, available for copying.
+*Object type* = Template, envelope, user and others listed above.
+*Mailbox* = Mailbox name and its UUID, available for copying.
+*User* = User name and its UUID, available for copying.
+*IP Address* = your IP address.
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst:172]: it's better to replace "your IP address" with "user's IP address"

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+*System* = Browser and OS.
+*Data* = Date when the event occurred.
+
+Envelope acion description
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst:176]: typo in word "acion"

**Review by kolapse2 — CHANGES_REQUESTED** [review]
Please make changes

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst (hunk 4 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst (hunk 3 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst (hunk 2 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst (hunk 1 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/auditLog/auditLog.rst]: done

**Review by kolapse2 — APPROVED** [review]
Approved

**Pull request merged**